sw/device/tests/sim_dv/entropy_src_fuse_en_fw_read_test.c

```diff
@@ -0,0 +1,197 @@
+// Copyright lowRISC contributors.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+
+#include "sw/device/lib/base/memory.h"
+#include "sw/device/lib/base/mmio.h"
+#include "sw/device/lib/dif/dif_base.h"
+#include "sw/device/lib/dif/dif_entropy_src.h"
+#include "sw/device/lib/dif/dif_rstmgr.h"
+#include "sw/device/lib/runtime/log.h"
+#include "sw/device/lib/testing/otp_ctrl_testutils.h"
+#include "sw/device/lib/testing/rstmgr_testutils.h"
+#include "sw/device/lib/testing/test_framework/check.h"
+#include "sw/device/lib/testing/test_framework/ottf_main.h"
+
+#include "hw/top_earlgrey/sw/autogen/top_earlgrey.h"  // Generated.
+#include "otp_ctrl_regs.h"                            // Generated
+
+OTTF_DEFINE_TEST_CONFIG();
+
+/**
+ * OTP HW partition relative IFETCH offset in bytes.
+ *
+ * x = OTP_CTRL_PARAM_EN_SRAM_IFETCH_OFFSET (1728)
+ * y = OTP_CTRL_PARAM_HW_CFG_OFFSET (1664)
+ * IFETCH_OFFSET = (x - y) = 64
+ */
+static const uint32_t kOtpIfetchHwRelativeOffset =
+    OTP_CTRL_PARAM_EN_SRAM_IFETCH_OFFSET - OTP_CTRL_PARAM_HW_CFG_OFFSET;
+
+static const uint32_t kOtpentropySrcFwReadOffset =
```

Comment:
nit: s/kOtpentropySrcFwReadOffset/kOtpEntropySrcFwReadOffset/

Reply:
I'm sorry, I didn't get it. Both search and replace names are the same.

Reply:
Entropy is capitalized :)

Reply:
ahh, how could I miss that x)!

Done